workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/services/tickets.ts

```diff
@@ -123,3 +125,48 @@ export async function generateNewTicket() {
 		method: 'POST',
 	});
 }
+
+export async function updateTicketStatus(ticket: Ticket) {
+	let listTypeDefinitions = {} as ListTypeDefinitions;
+
+	if (!(J3Y7_PRIORITIES in listTypeDefinitions)) {
+		listTypeDefinitions = await fetchListTypeDefinitions();
+	}
+	const priorities = listTypeDefinitions[J3Y7_PRIORITIES] as any[];
```

Comment:
Add a new line break before priorities.

Reply:
Also the Type `any[]` isn't very helpful. Can you be more specific with the type?